src/libs/actions/IOU.js

```diff
@@ -1271,6 +1290,51 @@ function payMoneyRequest(paymentType, chatReport, iouReport) {
     }
 }
 
+/**
+ * Initialize money request info and navigates to the MoneyRequest page
+ * @param {String} iouType
+ * @param {String} reportID
+ */
+function startMoneyRequest(iouType, reportID = '') {
+    resetMoneyRequestInfo(`${iouType}${reportID}`);
```

Comment:
Do we need to call this when we start a money request? It looks like we always call resetMoneyRequestInfo whenever we complete a command so is this redundant?

Reply:
I think we still need this if you abandon the money request (close money request modal without completing the process)

Reply:
But then at that point don't you want to continue the money request? I guess this is a NAB though

Reply:
I think we always want to clear it when we open it from FAB or Composer. We're following the same pattern with Assign Task.